src/modules/cmdpal/Microsoft.CmdPal.UI/MainWindow.xaml.cs

```diff
@@ -216,7 +196,7 @@ private static DesktopAcrylicController GetAcrylicConfig(UIElement content)
 
     private void ShowHwnd(IntPtr hwndValue, MonitorBehavior target)
     {
-        var hwnd = new HWND(hwndValue);
+        var hwnd = new HWND(hwndValue != 0 ? hwndValue : _hwnd);
```

Comment:
I don't like this.. 😒 
Do we really need to propagate CmdPal window HWND from various messages? Isn't the HWND fixed?